frontend/app/src/components/ui/token-input.tsx

```diff
@@ -0,0 +1,20 @@
+import { CopyToClipboard } from "../buttons/copy-to-clipboard";
+
+type TokenInputProps = {
+  value: string;
+};
+
+export const TokenInput = (props: TokenInputProps) => {
```

Comment:
TokenInput is not a "primitive" component. Thus, imo it does not belong to /ui folder